src/Cassandra/Metadata.cs

```diff
@@ -422,16 +422,84 @@ internal void ClearAggregate(string keyspaceName, string aggregateName, string[]
             }
         }
 
+        /// <summary>
+        /// Initiates a schema agreement check. This method does not perform retries so
+        /// <see cref="ProtocolOptions.MaxSchemaAgreementWaitSeconds"/> does not apply. 
+        /// </summary>
+        /// <returns>True if schema agreement was successful and false if it was not successful.</returns>
+        /// <exception cref="OperationCanceledException">If operation was canceled due to
+        /// the provided <see cref="CancellationToken"/></exception>
+        public async Task<bool> CheckSchemaAgreementAsync(CancellationToken ct = default(CancellationToken))
```

Comment:
I think providing cancellation options here seems like overkill, as the method only executes 2 queries in parallel that should take less than 1ms on a healthy cluster and returns the result. Additionally, we will be introducing the first driver method with a cancellation token that from a user standpoint it might seem arbitrary.

Reply:
Hmm I agree, the introduction of `CancellationToken` objects should be part of a larger initiative or release where it is introduced in multiple places of the API since it is not particularly relevant in this specific method. 

I'll remove the `CancellationToken` parameter but I'll leave the cancellation helper methods in there for future use, if that's ok.

Reply:
Sounds good.